packages/netease-cloud-music/default.nix

```diff
@@ -1,70 +1,445 @@
+    rm opt/netease/netease-cloud-music/libs/libnettle.so.6
+    rm opt/netease/netease-cloud-music/libs/libnfs.so.11
+    rm opt/netease/netease-cloud-music/libs/libnotify.so.4
+    rm opt/netease/netease-cloud-music/libs/libopenjp2.so.7
+    rm opt/netease/netease-cloud-music/libs/libopus.so.0
+    rm opt/netease/netease-cloud-music/libs/libpulse.so.0
+    # rm opt/netease/netease-cloud-music/libs/libqcef.so.1
+    # rm opt/netease/netease-cloud-music/libs/libqcef.so.1.1.4
+    # rm opt/netease/netease-cloud-music/libs/libQt5Core.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5DBus.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5EglFSDeviceIntegration.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5Gui.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5Network.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5Qml.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5Svg.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5WebChannel.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5Widgets.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5X11Extras.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5XcbQpa.so.5
+    # rm opt/netease/netease-cloud-music/libs/libQt5Xml.so.5
+    rm opt/netease/netease-cloud-music/libs/libraw1394.so.11
+    rm opt/netease/netease-cloud-music/libs/librom1394.so.0
+    rm opt/netease/netease-cloud-music/libs/libsamba-errors.so.1
+    rm opt/netease/netease-cloud-music/libs/libsamplerate.so.0
+    rm opt/netease/netease-cloud-music/libs/libsmime3.so
+    rm opt/netease/netease-cloud-music/libs/libsoxr.so.0
+    rm opt/netease/netease-cloud-music/libs/libspeexdsp.so.1
+    rm opt/netease/netease-cloud-music/libs/libspeex.so.1
+    rm opt/netease/netease-cloud-music/libs/libtalloc.so.2
+    rm opt/netease/netease-cloud-music/libs/libtasn1.so.6
+    rm opt/netease/netease-cloud-music/libs/libva-drm.so.2
+    rm opt/netease/netease-cloud-music/libs/libva.so.2
+    rm opt/netease/netease-cloud-music/libs/libva-wayland.so.2
+    rm opt/netease/netease-cloud-music/libs/libva-x11.so.2
+    rm opt/netease/netease-cloud-music/libs/libvdpau.so.1
+    rm opt/netease/netease-cloud-music/libs/libwacom.so.2
+    rm opt/netease/netease-cloud-music/libs/libxml2.so.2
+    rm opt/netease/netease-cloud-music/libs/libunistring.so.2
+    rm opt/netease/netease-cloud-music/libs/libUsageEnvironment.so.3
+    rm opt/netease/netease-cloud-music/libs/libxvidcore.so.4
+    rm opt/netease/netease-cloud-music/libs/libzvbi.so.0
+    rm opt/netease/netease-cloud-music/libs/libtdb.so.1
+    rm opt/netease/netease-cloud-music/libs/libtheoradec.so.1
+    rm opt/netease/netease-cloud-music/libs/libtheoraenc.so.1
+    rm opt/netease/netease-cloud-music/libs/libtwolame.so.0
+    rm opt/netease/netease-cloud-music/libs/libtheora.so.0
+    # rm opt/netease/netease-cloud-music/libs/libudev.so.1
+    # rm opt/netease/netease-cloud-music/libs/libXtst.so.6
```

Comment:
Maybe these tons of `rm`s can be refactored to `rsync ... --include essential_so_pattern` in `installPhase`?

Reply:
I'll take a try.